aws/resource_aws_wafregional_web_acl.go

```diff
@@ -0,0 +1,231 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/waf"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsWafRegionalWebAcl() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsWafRegionalWebAclCreate,
+		Read:   resourceAwsWafRegionalWebAclRead,
+		Update: resourceAwsWafRegionalWebAclUpdate,
+		Delete: resourceAwsWafRegionalWebAclDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"default_action": &schema.Schema{
+				Type:     schema.TypeSet,
+				Required: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": &schema.Schema{
+							Type:     schema.TypeString,
+							Required: true,
+						},
+					},
+				},
+			},
+			"metric_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"rules": &schema.Schema{
```

Comment:
As discussed in other regional WAF PRs - do you mind changing this field to singular `rule`?